routers/web/auth/oauth_test.go

```diff
@@ -66,25 +66,7 @@ func TestNewAccessTokenResponse_OIDCToken(t *testing.T) {
 
 	// Scopes: openid profile email
 	oidcToken = createAndParseToken(t, grants[0])
-	assert.Equal(t, user.Name, oidcToken.Name)
```

Comment:
Why delete these lines?

Reply:
Since the old code used `GetDisplayName`, the response would depend on the state of `setting.UI.DefaultShowFullName`, and it used to test both states. Now it's independent of that setting so only one set of checks is required.

Reply:
I don't think `assert.Equal(t, user.Website, oidcToken.Website)` will depends on `setting.UI.DefaultShowFullName`?

Reply:
Look at the lines below that change. They previously had a copy of the whole test. The lines I deleted have exact copies below.